### DISCUSSION TITLE
cn/2024/12/18/llm/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#197] [Announcements]
# cn/2024/12/18/llm/

https://hongtaoh.com/cn/2024/12/18/llm/

**Comment by earfanfan** [comment]
哇，你小子最近为撒耕了这么多博客啊，是因为学校放假了吗？

**Reply by hongtaoh** [reply]
最近也不知道为什么有很多想分享的。就写的比较多。

**Comment by earfanfan** [comment]
你家博客能不能加个功能，就是从这篇博客的页面可以跳转到上篇或下篇。

**Reply by hongtaoh** [reply]
寡人考虑一下。

**Reply by earfanfan** [reply]
你小子既然相信命理哲学，通晓八字星盘，那一定听说过紫微斗数吧。其实我悄悄推算过，你在甲辰龙年冬月二十日会变成一只青蛙，因为你声称自己为“寡人”。青蛙就是这么叫的：“孤寡孤寡”。

**Reply by hongtaoh** [reply]
刚把这个功能加上了。请过目。
